internal/gmm/handler.go

```diff
@@ -260,8 +265,6 @@ func CreatePDUSession(ulNasTransport *nasMessage.ULNASTransport,
 		gmm_message.SendDLNASTransport(ue.RanUe[anType], nasMessage.PayloadContainerTypeN1SMInfo,
 			smMessage, pduSessionID, cause, nil, 0)
 	} else {
-		ue.Lock.Lock()
```

Comment:
Why remove this lock?

It seems that the processing of N1N2message when creating a PDU session may fail due to the removal of this lock.